Src/StackifyLib/API.cs

```diff
@@ -33,16 +33,6 @@ public static Models.Device[] GetDeviceList()
             }
         }
 
-        public static bool SetServerEnvironmentForThisDevice(string environmentName)
```

Comment:
It isn't by us but potentially could be by our users. But I think it is unlikely.